### DISCUSSION TITLE
How to calculate "height" of an SVG?

### DISCUSSION THREAD

**Discussion by bitttttten** [fabricjs/fabric.js#10278] [Q&A]
We have an SVG like this:

```html
<svg xmlns="http://www.w3.org/2000/svg" width="32" height="16">
            <path stroke="black" fill="none" d="M 1 8 L 30 8" />
</svg>
```

Running it through `loadSVGFromString` gives us an object with width: 29 and height 0. This makes sense since the path has no height, it is using the default stroke width of 1.

You can see in this example: https://jsfiddle.net/e6vwtx1k/4/ the height is 0, strokeWidth is 1.

In short I am looking for a way to "edit" the "height" of an element, and this is where I am stuck. I understand I can sometimes do `height*scaleY` to get the relative height in the canvas. However how do I manage a svg path like this?

**Comment by asturur** [comment]
the myPath._getTransformedDimensions() should give you a correct size for bounding box of the object.

**Reply by asturur** [reply]
Why does it have that low dash in front of it? I do not know, it was always named like that and i didn't want to rename it to avoid breaking people code

**Comment by 752337625** [comment]
可以通过使用创建js Image对象，之后使用load获取图形高度和宽度信息。
